Cargo.toml

```diff
@@ -89,3 +89,8 @@ required-features = ["async-std"]
 name = "dbus_service"
 path = "examples/dbus_service.rs"
 required-features = ["async-std"]
+
+[workspace]
```

Comment:
doesn't sound good at all, i would rather move oo7 to a client sub-directory in a separate PR first and setup the workspace infrastructure before getting into such things

Reply:
Removed. It was a nice hack to avoid having one extra CI for rust fmt